sale_force_invoiced/__init__.py

```diff
@@ -1,3 +1,4 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 from . import model
+from . import wizard
```

Comment:
```suggestion
from . import wizards
```

See https://github.com/OCA/maintainer-tools/tree/master/template/module